src/api/feed-discovery/src/middleware.js

```diff
@@ -2,16 +2,25 @@ const got = require('got');
 const cheerio = require('cheerio');
 const { logger, createError } = require('@senecacdot/satellite');
 
+// check if the URL is valid.
+const isUrl = (url) => {
+  try {
+    return new URL(url);
+  } catch {
+    throw createError(400, 'Invalid Blog URL');
```

Comment:
Why don't you return `false` here, then throwing `400, 'Invalid Blog URL'` on line 23?